platform-core-module/src/main/java/com/softicar/platform/core/module/event/AGSystemEvent.sqml

```diff
@@ -0,0 +1,14 @@
+PACKAGE com.softicar.platform.core.module.event
+
+IMPORT com.softicar.platform.core.module.event.severity.AGSystemEventSeverity
+IMPORT com.softicar.platform.core.module.user.AGUser
+
+TABLE AGSystemEvent "Core.SystemEvent" {
+	PK Integer id = SERIAL
+	Boolean needsAttention
```

Comment:
I think the consistency in our terminology could be improved:
The button labelled `Confirm All` flips the `Needs Attention` flag. That's not intuitive - we're using two different terms for the same concept.
I therefore suggest to rename this column to `Needs Confirmation`.

Reply:
as you desire :)